client.go

```diff
@@ -240,6 +240,13 @@ func (c *Client) SetBootDevice(ctx context.Context, bootDevice string, setPersis
 	return ok, err
 }
 
+// SetVirtualMedia pass through to library functino
```

Comment:
From the client consumers perspective, what does calling this do? I think that'd be helpful to outline in the method comment.